tests/tests/tier0/bluechi-agent-config-c-option/test_agent_config_c_option.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import os
+import logging
+
+from typing import Dict
+from bluechi_test.test import BluechiTest
+from bluechi_test.container import BluechiControllerContainer, BluechiNodeContainer
+from bluechi_test.config import BluechiControllerConfig, BluechiNodeConfig
+from bluechi_test.util import read_file
+
+LOGGER = logging.getLogger(__name__)
+
+NODE_FOO = "node-foo"
+active_status = "active"
+failed_status = "failed"
+
+
+def restart_with_config_file(config_file_location, container, service):
```

Comment:
This method is used in multiple tests, so it's probably worth to add to `BluechiContainer` class in `bluechi_test/container.py`